lib/ffi-icu/version.rb

```diff
@@ -1,3 +1,3 @@
 module ICU
-  VERSION = "0.5.3"
+  VERSION = '0.5.3'.freeze
```

Comment:
Adding the `# frozen_string_literal: true` comment could help. Related [cop](https://docs.rubocop.org/rubocop/cops_style.html#stylefrozenstringliteralcomment).